dali/CMakeLists.txt

```diff
@@ -70,6 +70,20 @@ install(TARGETS ${dali_lib} DESTINATION lib)
 install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR} DESTINATION
   include FILES_MATCHING PATTERN "*.h" PATTERN "*_test.h" EXCLUDE)
 
+# install libdali to build dir
+add_custom_command(
+    TARGET ${dali_lib} POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy
+            $<TARGET_FILE:${dali_lib}>
+            ${CMAKE_BINARY_DIR}/lib/$<TARGET_FILE_NAME:${dali_lib}>)
```

Comment:
Should we just change the target location?
Like this:
```
set_target_properties(mylibrary PROPERTIES
RUNTIME_OUTPUT_DIRECTORY_DEBUG <debug path>
RUNTIME_OUTPUT_DIRECTORY_RELEASE <release path>
)
```
Then again this might be the only case in this PR where this applies, so might not be worth having two sorts of idioms for that.  ?

Reply:
Done